src/components/elements/market-tab/index.tsx

```diff
@@ -27,7 +27,7 @@ const MarketTab = () => {
         <>
             <MarketsContainer>
                 {market_buttons.map((marketItem) => (
-                    <>
+                    <React.Fragment key={marketItem.market_name}>
                         <MarketButton
                             selected={marketItem.market_name === selected_market}
                             key={marketItem.id}
```

Comment:
while we have this line, are we need `<React.Fragment key={marketItem.market_name}>`?

Reply:
I missed to delete the `key` from this line. I updated the line 30 because it was having console error stating map should have key